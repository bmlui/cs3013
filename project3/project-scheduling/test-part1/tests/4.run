./scheduler 0 SJF tests/4.in 0
