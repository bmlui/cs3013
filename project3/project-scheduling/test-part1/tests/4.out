Execution trace with SJF:
t=0: [Job 0] arrived at [0], ran for: [5]
End of execution with SJF.
